packages/editor-ui/src/components/RunData.vue

```diff
@@ -288,6 +288,11 @@ export default defineComponent({
 				return false;
 			}
 
+			if (this.outputIndex !== 0) {
```

Comment:
Would it be more safe here to check the type instead of the output index? Because, for example, if the UX changes and suddenly error output is on top, this would be wrong.

Reply:
Indeed this assumption is a bit wacky. It's rooted in the explicit effort elsewhere in the code to add the error path *after* node specific outputs, but it might not hold long-term.

So it turns out we actually don't need this check here, as this line https://github.com/n8n-io/n8n/blob/73b0a80ac92b4f4b5a300d0ec1c833b4395a222a/packages/editor-ui/src/components/RunData.vue#L263 has `!this.rawInputData.length`, which causes us to return early for errors.
I don't fully understand this, does this happen to align, or can we rely on this semantically?

I can still add the check, though I'd move it to `canPinNode` by adding an index parameter if that works for you.